Fixed a bug where the canonical chain was changed during preload (#862)
planetarium/libplanet: CHANGES.md

### Bug fixes

>  -  Fixed a bug where the canonical chain was changed if executing actions are failed during
>     preload.  [[#862]]

Reviewer: ```suggestion
 -  Fixed a bug where the canonical chain had changed if any actions had thrown an exception
    during `Swarm<T>.PreloadAsync()`.  [[#862]]
```
